src/pages/registration/DescriptionPanel.tsx

```diff
@@ -136,7 +136,7 @@ const DescriptionPanel = () => {
         </DateAndLanguageWrapper>
       </BackgroundDiv>
       <BackgroundDiv backgroundColor={lightTheme.palette.section.dark}>
-        <Typography variant="h5" color="primary">
+        <Typography variant="h5" color="primary" variantMapping={{ h5: 'body1' }}>
```

Comment:
dette blir vel en `<body1>`-tag, som egentlig ikke er noe?
```suggestion
        <Typography variant="h5" color="primary" component="h2">
```
evt `component="p"`

Reply:
`body1 = p`. https://material-ui.com/api/typography/

Reply:
for variant ja, men mapping sier hvilken HTML-tag som skal brukes, så dette blir `<body1>` i DOM som ser ut som en `h5`, men det er ingenting som heter `body1` i HTML. I linken der ser du at man skal mappe til HTML-tags, feks ved at "body1" skal være "p"

Reply:
ohh, da har jeg misforstått! takk!